themes/_core/js/cart.js

```diff
@@ -28,8 +28,8 @@ import {refreshCheckoutPage} from './common';
 
 $(document).ready(() => {
   prestashop.on('updateCart', (event) => {
-    prestashop.cart = event.reason.cart;
-    let getCartViewUrl = $('.js-cart').data('refresh-url');
+    prestashop.cart = event.resp.cart;
+    var getCartViewUrl = $('.js-cart').data('refresh-url');
```

Comment:
```suggestion
    let getCartViewUrl = $('.js-cart').data('refresh-url');
```
This change wasn't needed

Reply:
Totally true. What a shame!!! I am for sure a newbie in branching and resolving conflicts :).
I maybe polluted this second PR.

Reply:
I think it is ok now. The PR seems to include only the my code now. wdyt?

Reply:
:( But a PR with 13 commits for 3 lines of code === non sense. I will try again

Reply:
You can use `git rebase -i develop` to squash your commits :wink: